tests/test_databag.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+import os
+
+from lektor.builder import Builder
+from lektor.db import Database
+from lektor.environment import Environment
+from lektor.project import Project
+
+
+def get_databag_builder(tmpdir):
```

Comment:
As a pytest style note, this could be structured as a fixture.
```py
@pytest.fixture
def databag_pad():
    project = Project.from_path(Path(__file__).parent / "databag-project")
    return project.make_env(load_plugins=False).new_pad()

@pytest.fixture
def databag_builder(databag_pad, tmp_path):
    return Builder(databag_pad, tmp_path / "output")

def test_databag_project_folder(databag_pad, databag_builder):
    ...
```

Or, perhaps, just override the `project` fixture (defined in `conftest.py`) which would allow for the direct use of the `pad` and `builder` fixtures from `conftest.py`:
```py
@pytest.fixture
def project():
    return Project.from_path(Path(__file__).parent / "databag-project")

def test_databag_project_folder(pad, builder):
    ...
```